This patch will not be necessary once https://github.com/karalabe/usb/pull/14 has been merged.

diff -u -r ../go-ethereum-1.9.9/accounts/usbwallet/hub.go ./accounts/usbwallet/hub.go
--- ../go-ethereum-1.9.9/accounts/usbwallet/hub.go	2019-12-06 09:51:37.000000000 +0000
+++ ./accounts/usbwallet/hub.go	2019-12-12 00:16:14.379913000 +0000
@@ -26,7 +26,7 @@
 	"github.com/ethereum/go-ethereum/accounts"
 	"github.com/ethereum/go-ethereum/event"
 	"github.com/ethereum/go-ethereum/log"
-	"github.com/karalabe/usb"
+	"github.com/fornwall/usb"
 )
 
 // LedgerScheme is the protocol scheme prefixing account and wallet URLs.
diff -u -r ../go-ethereum-1.9.9/accounts/usbwallet/wallet.go ./accounts/usbwallet/wallet.go
--- ../go-ethereum-1.9.9/accounts/usbwallet/wallet.go	2019-12-06 09:51:37.000000000 +0000
+++ ./accounts/usbwallet/wallet.go	2019-12-12 00:16:26.669913000 +0000
@@ -31,7 +31,7 @@
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/ethereum/go-ethereum/log"
-	"github.com/karalabe/usb"
+	"github.com/fornwall/usb"
 )
 
 // Maximum time between wallet health checks to detect USB unplugs.
diff -u -r ../go-ethereum-1.9.9/go.mod ./go.mod
--- ../go-ethereum-1.9.9/go.mod	2019-12-06 09:51:37.000000000 +0000
+++ ./go.mod	2019-12-12 00:09:11.128276000 +0000
@@ -33,7 +33,7 @@
 	github.com/influxdata/influxdb v1.2.3-0.20180221223340-01288bdb0883
 	github.com/jackpal/go-nat-pmp v1.0.2-0.20160603034137-1fa385a6f458
 	github.com/julienschmidt/httprouter v1.1.1-0.20170430222011-975b5c4c7c21
-	github.com/karalabe/usb v0.0.0-20190919080040-51dc0efba356
+	github.com/fornwall/usb v0.0.0-20191212001557-3a1f5af8866c
 	github.com/kr/pretty v0.1.0 // indirect

 	github.com/kylelemons/godebug v1.1.0 // indirect
 	github.com/mattn/go-colorable v0.1.0
